biz.aQute.resolve/test/biz/aQute/resolve/RunResolutionTest.java

```diff
@@ -108,7 +108,10 @@ public void testNotCachingOfResultForOtherResolveOption() throws Exception {
 		Collection<Container> runbundles = pl.getProject()
 			.getRunbundles();
 		assertThat(runbundles).isEmpty();
-		assertThat(bndrun.check("Download java.io.FileNotFoundException:")).isTrue();
+		if (IO.isWindows())
+			assertThat(bndrun.check()).isTrue();
```

Comment:
Why do we not have the same result on Winwdows? We can find the file on windows but not on *ix?

Reply:
That is what  I am trying to figure out :-(